apps/history/declarativehistorymodel.cpp

```diff
@@ -100,6 +104,8 @@ QVariant DeclarativeHistoryModel::data(const QModelIndex & index, int role) cons
         return url.title().isEmpty() ? url.url() : url.title();
     case DateRole:
         return url.date();
+    case FaviconRole:
+        return FaviconManager::instance()->get("history", url.url());
```

Comment:
Should this be a `QStringLiteral("history")`? I always seem to misjudge the correct optimisation with this.

Reply:
done